core-service/src/main/java/com/crmfoodestablishment/coreservice/service/MenuService.java

```diff
@@ -14,36 +16,49 @@ public class MenuService {
 
     private final MenuRepository menuRepository;
 
+    private final MenuMapper menuMapper;
+
     private Menu checkIfMenuExist(Integer id) {
```

Comment:
Single responsibility principle is violated. Even if you want to keep such way, consider change name to something like checkAndFindMenuById

Reply:
In the comments above, Polina write that it would be better to name the method checkIfMenuExist. In the end, I don't really care), I can call it that if everyone is in favour.